ApplyToBecomeInternal/ApplyToBecomeInternal/Models/ProjectViewModel.cs

```diff
@@ -23,6 +25,19 @@ public ProjectViewModel(Project project)
 			AssignedDate = FormatDate(project.AssignedDate);
 			Phase = _projectPhaseText[project.Phase];
 		}
+		public ProjectViewModel(Project project, string section)
+		{
+			Id = project.Id.ToString();
```

Comment:
Copying the constructor body is redundant. You should use [constructor chaining](https://www.c-sharpcorner.com/UploadFile/825933/constructor-chaining-in-C-Sharp/)